### PR TITLE
Open/Close feature

### PR DESCRIPTION
**Pull request by rachids**
- Admin can close/open a place.
- Added a bunch of classes to be able to filter easily based on query params

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -41,10 +44,11 @@
 Auth::routes(['register' => false]);
 Route::get('/categorie/{category:slug}', 'CategoryController@index')->name('categories.index');
 
+Route::get('/region/{region:slug}', 'PublicController@indexRegional')->name('public.index-region');
 Route::get('/region/province', 'PublicController@indexProvincial')->name('public.index-provincial');
 Route::get('/region/{region:slug}', 'PublicController@indexRegional')->name('public.index-region');
 Route::get('/region/{region:slug}/{category}', 'PublicController@indexRegionalCategories')->name('public.index-region-category');
 Route::get('/entreprise/ajout', 'PlaceController@createPublic')->name('places.create-public');
-Route::post('/entreprise/ajout', 'PlaceController@storePublic')->name('places.store-public')->middleware(ProtectAgainstSpam::class);
```
Comment (poliquinp) [inline routes/web.php]: Forget to reactivate the middleware.

Comment (rachids) [reply routes/web.php]: Oops, my bad. I had to remove it so I could naviguate locally.
Will fix this.

**Review by jpmurray — APPROVED** [review]

**Comment by jpmurray** [comment]
Merging this in its own branch for further testing and getting latest develop changes, but should be merged soon enough to develop than master!

File: app/Place.php [hunk 2]
```diff
@@ -58,6 +61,11 @@ public function getPageTitle(): string
         return "{$this->name}{$midName}".config('app.name', '');
     }
 
+    #SCOPES
+    public function scopeOpened($query)
+    {
+        return $query->where('is_closed', 0);
+
```
Comment (jpmurray) [inline app/Place.php:68]: Just as feedback, but you where missing a closing bracket here ;-)

Comment (rachids) [reply app/Place.php:68]: Resolving conflict within Github gone wrong :s 
Thanks for fixing it!

**Pull request merged**